app/lib/asyncstorage-chunk-store.js

```diff
@@ -72,8 +72,21 @@ Storage.prototype.get = function(index, opts, cb) {
 
 Storage.prototype.close = Storage.prototype.destroy = function(cb) {
```

Comment:
should we also make `close` a separate method that just sets `this.closed = true`?